### DISCUSSION TITLE
Error messages

### DISCUSSION THREAD

**Discussion by GitHubDan01** [tsunamods-codes/7th-Heaven#194] [Q&A] [answered]
When starting the mod manager i am given an update error message that doesn't tell me how to fix 

I have the mods i want to use loaded and as soon as i click to play the game it opens then 5 seconds later gamecrash:( 

I dont know what to do and i have tried the steps it says and it just leads me to a page where theres no info

**Comment by Orteku** [comment]
I have the same problem with the last canary update, it gives me this error:
![xd5YQAb](https://github.com/tsunamods-codes/7th-Heaven/assets/53489000/57b1e7af-5d9a-4023-914e-d50ee9a44fa6)

**Comment by julianxhokaxhiu** [comment, marked as answer]
Please join the Discord server and see the relative announcements (you can find the link here: https://7thheaven.rocks/#download7h). Canaries are undergoing a new change that will make the 7th Heaven even more easier to use, but we need to test some aspects of it. If you're willing to contribute hop on Discord so we can have an easier troubleshooting. Thanks.
